src/test/java/com/epam/hw2/ExerciseFirst.java

```diff
@@ -0,0 +1,98 @@
+package com.epam.hw2;
+
+import org.openqa.selenium.*;
+import org.testng.annotations.Test;
+import java.util.concurrent.TimeUnit;
+
+
+public class ExerciseFirst extends AbstractWebPageTest {
+
+    @Test(groups = {"group-hw2"}, priority = 1)
+    public void exerciseFirstTest() {
+        webDriver().manage().window().maximize();
+        webDriver().manage().timeouts()
+                .implicitlyWait(2, TimeUnit.SECONDS);
+        // 1 | Open test site by URL
+        webDriver().navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        // 2 | Assert Browser title
+        asert().assertEquals(webDriver().getTitle(), "Home Page");
+        // 3 | Perform login username: Roman  pass: Jdi1234
+        webDriver().findElement(By.xpath("//a[@href='#']")).click();
+        webDriver().findElement(By.id("name")).sendKeys("Roman");
+        webDriver().findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver().findElement(By.id("login-button")).click();
+        webDriver().manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+        // 4 | Assert Username is loggined "ROMAN IOVLEV" Name is displayed
+        asert().assertEquals("ROMAN IOVLEV", webDriver()
+                                                    .findElement(By.id("user-name")).getText());
+        // 5 | Assert that there are 4 items on the header section are displayed
+        //!"HOME", "CONTACT FORM", "SERVICE", "METALS & !COLORS"
+        asert().assertEquals(webDriver()
+                                .findElement(By.xpath("//div[@class='uui-header dark-gray']"
+                                                       + "//*//a[@href='index.html']")).getText(),
+                                "HOME");
+        asert().assertEquals(webDriver()
+                                .findElement(By.xpath("//div[@class='uui-header dark-gray']"
+                                                    + "//*//a[@href='contacts.html']")).getText(),
+                                "CONTACT FORM");
+        asert().assertEquals(webDriver()
+                                .findElement(By.xpath("//div[@class='uui-header dark-gray']"
+                                                  + "//*//a[@class='dropdown-toggle']")).getText(),
+                                "SERVICE");
+        asert().assertEquals(webDriver()
+                                .findElement(By.xpath("//div[@class='uui-header dark-gray']"
+                                                + "//*//a[@href='metals-colors.html']")).getText(),
+                                "METALS & COLORS");
+        // 6 | Assert that there are 4 images on the Index
+        asert().assertEquals(webDriver()
+                            .findElements(By.xpath("//div[@class='benefit-icon']")).size(),
```

Comment:
Bys' priority:
1. DOM (id, name)
2. CSS 
3. XPATH

```suggestion
                            .findElements(By.css("div.benefit-icon")).size(),
```